SConstruct

```diff
@@ -269,8 +280,7 @@ build_scripts=[
     'share/rubygems/SConstruct',
     'src/im_mad/collectd/SConstruct',
     'src/client/SConstruct',
-    'src/docker_machine/SConstruct',
-    svncterm_path
```

Comment:
needed

Reply:
true, my fault since i remove it for mac compilation